lib/content-services/src/lib/site-dropdown/sites-dropdown.component.ts

```diff
@@ -77,6 +77,9 @@ export class DropdownSitesComponent implements OnInit {
     @Output()
     change: EventEmitter<SiteEntry> = new EventEmitter();
 
+    @Output()
+    error = new EventEmitter<any>();
```

Comment:
Does this need to be `any`?

Reply:
Oh and btw shouldn't those new outputs be documented per each component?

Reply:
we can generate docs later, I'm just removing the logging service, which was not documented either